imgee/models/stored_file.py

```diff
@@ -58,7 +59,34 @@ def __repr__(self):
     def extn(self):
         return guess_extension(self.mimetype, self.orig_extn) or ''
 
-    def is_queued_for_deletion(self):
-        if imgee.app.config.get('CELERY_ALWAYS_EAGER'):
-            return False
-        return imgee.registry.is_queued_for_deletion(self.name+self.extn)
+    @property
+    def filename(self):
+        return self.name + self.extn
+
+    def dict_data(self):
+        return dict(
+            title=self.title,
+            uploaded=self.created_at.isoformat() + 'Z',
+            filesize=app.jinja_env.filters['filesizeformat'](self.size),
```

Comment:
Looks like it needs a `binary` parameter too. https://github.com/pallets/jinja/blob/master/jinja2/filters.py#L459